### PR TITLE
Fix server/tests persistence - MySQL table name

### PR DESCRIPTION
**Pull request by gustavolira**
https://issues.redhat.com/browse/ISPN-13993

### CODE REVIEW

File: server/tests/src/test/java/org/infinispan/server/persistence/TableManipulation.java [hunk 1]
```diff
@@ -30,11 +31,14 @@ public class TableManipulation implements AutoCloseable {
...
    private static final String DEFAULT_IDENTIFIER_QUOTE_STRING = "\"";
    private static final String TABLE_NAME_PREFIX = "tbl";
-   private final String tableName;
+   private String tableName;
 
    public TableManipulation(String cacheName, PooledConnectionFactoryConfigurationBuilder persistenceConfiguration, ConfigurationBuilder configurationBuilder) {
       this.persistenceConfiguration = persistenceConfiguration;
       this.tableName = String.format("%s%s_%s%s", DEFAULT_IDENTIFIER_QUOTE_STRING, TABLE_NAME_PREFIX, cacheName, DEFAULT_IDENTIFIER_QUOTE_STRING);
+      if (isMysql()) {
+         this.tableName = this.tableName.replaceAll("\"", "");
```
Comment (ryanemerson) [inline server/tests/src/test/java/org/infinispan/server/persistence/TableManipulation.java:40]: In our production code we use ` as the identifier quote string, shouldn't it be the same here?

**Review by ryanemerson — COMMENTED** [review]
@gustavolira How come this change is necessary now, have the MySQL tests always failed?

**Comment by wburns** [comment]
@gustavolira do you have a JIRA for this PR?

File: server/tests/src/test/java/org/infinispan/server/persistence/TableManipulation.java (hunk 1 above)
Comment (gustavolira) [reply server/tests/src/test/java/org/infinispan/server/persistence/TableManipulation.java:40]: @ryanemerson AbstractTableManager use the same " as the identifier quote string

**Comment by gustavolira** [comment]
> @gustavolira do you have a JIRA for this PR?

Do we need a JIRA for it? it's just a simple fix on test

**Comment by wburns** [comment]
> > @gustavolira do you have a JIRA for this PR?
> 
> @wburns Do we need a JIRA for it? it's just a simple fix on test

Yes, having a JIRA with a description saying what the bug is needed, even for tests.

**Comment by gustavolira** [comment]
> > > @gustavolira do you have a JIRA for this PR?
> > 
> > 
> > @wburns Do we need a JIRA for it? it's just a simple fix on test
> 
> Yes, having a JIRA with a description saying what the bug is needed, even for tests.

Updated with the JIRA https://issues.redhat.com/browse/ISPN-13993

**Comment by tristantarrant** [comment]
Related test failure: testFailoverWithPassivation[mysql] – org.infinispan.server.persistence.JdbcStringBasedCacheStorePassivation

**Comment by tristantarrant** [comment]
Merged, thanks

File: server/tests/src/test/java/org/infinispan/server/persistence/TableManipulation.java (hunk 1 above)
Comment (ryanemerson) [reply server/tests/src/test/java/org/infinispan/server/persistence/TableManipulation.java:40]: MySQL uses the `MyTableOperations` implementation which has the ` identifier

**Pull request merged**